htdocs/compta/tva/card.php

```diff
@@ -333,6 +333,10 @@
 		$object->id = $object->ref = null;
 		$object->paye = 0;
 
+		if (GETPOST('amount', 'alphanohtml')) {
```

Comment:
Must be GETPOSTISSET instead of GETPOST so we can set amount to 0

Reply:
Hi @eldy, on vat and salaries amount is mandatory, why do you want to be able to set it to 0 ?
![image](https://user-images.githubusercontent.com/5585819/203502255-7265475e-c993-4d0a-871c-3d023dfc9802.png)

Reply:
I used the same test than social charges, but in this one there is a "$object->check();" before create, then if amount is null, clone action fails

Reply:
@eldy did you see my answer ?